frontend/src/metabase/query_builder/components/DimensionList.jsx

```diff
@@ -102,7 +105,7 @@ export default class DimensionList extends Component {
         );
 
     return (
-      <div className="Field-extra flex align-center">
+      <DimensionListSubDimensionContainer className="Field-extra">
```

Comment:
I haven't tested this component, but I think it should be able to be replaced with either Mantine's [`Flex`](https://mantine.dev/core/flex/) or [`Box`](https://mantine.dev/core/box/) component.